### PR TITLE
Parameterize run image. Don't pull during pack phase if using dev.local

### LINKED ISSUE #956: pack build should use no-pull==true when using dev.local

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #956

### CODE REVIEW

**Comment by ericbottard** [comment]
This builds on top of #954 

Did not make the run image an actual flag, but we are one line of code away from it

File: cmd/commands/wiring.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func resolveHomePath(p string) (string, error) {
 
 }
 
-func CreateAndWireRootCommand(manifests map[string]*core.Manifest, builder string) *cobra.Command {
+func CreateAndWireRootCommand(manifests map[string]*core.Manifest, localBuilder string, defaulRunImage string) *cobra.Command {
```
Comment (markfisher) [inline cmd/commands/wiring.go]: typo "defaulRunImage"

File: cmd/commands/wiring.go [hunk 2]
```diff
@@ -101,7 +101,7 @@ See https://projectriff.io and https://github.com/knative/docs`,
 	function := Function()
 	installKubeConfigSupport(function, &client, &kc)
 	function.AddCommand(
-		FunctionCreate(&client, builder),
+		FunctionCreate(&client, FunctionCreateDefaults{LocalBuilder: localBuilder, DefaultRunImage: defaulRunImage}),
```
Comment (markfisher) [inline cmd/commands/wiring.go]: typo "defaulRunImage"

**Comment by markfisher** [comment]
LGTM

fixed typos, rebased, squashed and merged via https://github.com/projectriff/riff/commit/2d33e1d414ae9c8ca7490ccd997a0d8be7d5f77d

**Pull request closed without merging**